checkstyle.xml

```diff
@@ -289,6 +289,12 @@
         <module name="CommentsIndentation">
             <property name="tokens" value="SINGLE_LINE_COMMENT, BLOCK_COMMENT_BEGIN"/>
         </module>
+        <module name="Indentation">
```

Comment:
why do you need this changes?

Reply:
Tried to fix the check for indentation for the last line

Reply:
removed the module